pkg/datapath/linux/route/route_linux.go

```diff
@@ -407,25 +407,6 @@ func ListRules(family int, filter *Rule) ([]netlink.Rule, error) {
 		return rules, nil
 	}
 
-	const (
-		// RT_FILTER_PRIORITY is a flag that can be specified to signal
-		// filtering against rules with 'Priority' specified. Note: this count
-		// starts from where netlink stops, see
-		// https://github.com/vishvananda/netlink/blob/d71301a47b607450337d920f260f3dc76481298e/route_linux.go#L25
-		//
-		// TODO: Remove this function when the upstream PR has been merged:
-		// https://github.com/vishvananda/netlink/pull/538
```

Comment:
Looks like this whole function can be replaced with the upstream version, as https://github.com/vishvananda/netlink/pull/538 was merged.

Reply:
Sorry - I didn't spot this in the first place. Updated the PR accordingly.